recipes/nest/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "nest-simulator" %}
+{% set version = "2.16.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nest/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: abfeb61719dec54da9477be035bef1d9d764f4e7663f63f6a6d9211f967e0490
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - cmake >=3.6
+    - git
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - cython
+    - docutils
+    - future
+    - gsl
+    - icu
+    - jsoncpp
+    - libtool
+    - matplotlib
```

Comment:
Are these all needed? Some of these like `pandas`, `matplotlib` look like runtime only dependencies.

Reply:
The requirements are now tidied up